portmidi.go

```diff
@@ -25,35 +25,10 @@ import (
 	"errors"
 )
 
-var (
-	ErrUnknown            = errors.New("portmidi: unknown error")
-	ErrNoData             = errors.New("portmidi: no data")
-	ErrHost               = errors.New("portmidi: host error")
-	ErrInvalidDeviceId    = errors.New("portmidi: invalid device id")
-	ErrInsufficientMemory = errors.New("portmidi: insufficient memory")
-	ErrBufferTooSmall     = errors.New("portmidi: buffer too small")
-	ErrBufferOverflow     = errors.New("portmidi: buffer overflow")
-	ErrBadPtr             = errors.New("portmidi: bad ptr")
-	ErrBadData            = errors.New("portmidi: bad data")
-	ErrInternalError      = errors.New("portmidi: internal error")
-	ErrBufferMaxSize      = errors.New("portmidi: buffer max size")
-)
-
-var errorMap map[int]error = map[int]error{
-	0:      nil,
-	-10000: ErrHost,
-	1:      ErrInvalidDeviceId,
-	2:      ErrInsufficientMemory,
-	3:      ErrBufferTooSmall,
-	4:      ErrBufferOverflow,
-	5:      ErrBadPtr,
-	6:      ErrBadData,
-	7:      ErrInternalError,
-	8:      ErrBufferMaxSize,
-}
-
+// DeviceId is a MIDI device ID.
 type DeviceId int
 
+// DeviceInfo provides info about a MIDI device.
 type DeviceInfo struct {
 	Interface         string
```

Comment:
Not too sure, but I think it's the underlying lib that portmidi uses, like "ALSA" or "CoreMIDI" or something like that (judging by [this](https://sourceforge.net/p/portmedia/code/HEAD/tree/portmidi/trunk/pm_common/portmidi.h#l209))